users/cfarvidson/cfarvidson.h

```diff
@@ -0,0 +1,29 @@
+#include QMK_KEYBOARD_H
+#include "keymap_swedish_pro_osx_ansi.h"
+
+#define _BASE 0
+#define _NAV 1
+#define _MOUSE 2
+#define _MEDIA 3
+#define _NUM 4
+#define _SYM 5
+#define _FUN 6
```

Comment:
we heavily recommend using an enum for the layers, as it is easier to maintain long term, and less likely to introduce accidental firmware bloat (empty/unused layers take up space, for instance).